docs/configuration/smime_email_signing.rst

```diff
@@ -0,0 +1,302 @@
+- :xref:`SparkPost S/MIME Technical Guide`
+
+How it works
+************
+
+When you turn on and properly configure S/MIME signing:
+
+1. Mautic signs each outgoing Email with the private key for the sender's Email address
+2. Mautic attaches the signature to the Email as a ``smime.p7s`` file
+3. Recipients with S/MIME-capable Email clients can verify the signature using the public certificate
+4. If no certificate exists for a sender's Email address, Mautic sends the Email unsigned without error
+
+Turn on S/MIME signing
+**********************
+
+Turn on S/MIME signing through your Mautic configuration file. It isn't available in the web interface.
+
+.. warning::
+    Make sure you're using the SMTP Email transport before turning on S/MIME signing. The feature doesn't work with API-based Email transports.
+
+Configuration
+=============
+
+Add the following configuration parameters to your ``app/config/local.php`` file:
+
+.. code-block:: php
+
+    <?php
+    $parameters = array(
+        // ... other configuration ...
+        'smime_signing_enabled' => true,
+        'smime_certificates_path' => '%kernel.project_dir%/var/smime_certificates',
+    );
+
+Configuration parameters
+------------------------
+
+``smime_signing_enabled``
+    set to ``true`` to turn on S/MIME Email signing or ``false`` to turn it off.
+    
+    Default: ``false``
+
+``smime_certificates_path``
+    the absolute path to the directory where Mautic stores your S/MIME certificates. You can use ``%kernel.project_dir%`` to reference your Mautic installation directory.
+    
+    Default: ``%kernel.project_dir%/var/smime_certificates``
+
+Generating S/MIME certificates
+*******************************
+
+Each Email address that sends Emails from Mautic needs its own pair of certificates:
+
+- A **public certificate** (``.crt`` file) that verifies your identity
+- A **private key** (``.pem`` file) that signs the Emails
+
+Self-signed certificates
+=========================
+
+For testing purposes, you can create self-signed certificates. However, for production use, you should obtain certificates from a trusted Certificate Authority.
+
+To create a self-signed certificate and private key:
+
+.. code-block:: bash
+
+    # Create a private key and certificate signing request
+    openssl req -newkey rsa:4096 -nodes -keyout sender@example.com.pem -out sender@example.com.csr
+    
+    # Create a self-signed certificate valid for 1 year
+    openssl x509 -req -days 365 -in sender@example.com.csr -signkey sender@example.com.pem -out sender@example.com.crt
+
+.. important::
+    Replace ``sender@example.com`` with the actual Email address you're using to send Emails from Mautic.
+
+Production certificates
+=======================
+
+For production use, obtain S/MIME certificates from a trusted Certificate Authority - CA. Many Certificate Authorities offer S/MIME certificates, and the process typically involves:
+
+1. Generating a Certificate Signing Request - CSR
```

Comment:
I think we can list CSR in `accept.txt`.

Reply:
Done